ci-operator/step-registry/sandboxed-containers-operator/env-cm/sandboxed-containers-operator-env-cm-commands.sh

```diff
@@ -72,9 +72,11 @@ if [[ "$TEST_RELEASE_TYPE" == "Pre-GA" ]]; then
   create_catsrc "${CATALOG_SOURCE_NAME}" "${CATALOG_SOURCE_IMAGE}"
   create_catsrc "${TRUSTEE_CATALOG_SOURCE_NAME}" "${TRUSTEE_CATALOG_SOURCE_IMAGE}"
 else
-  if [[ -n "$CATALOG_SOURCE_IMAGE" || -n "$TRUSTEE_CATALOG_SOURCE_IMAGE" ]]; then
-    echo "CATALOG_SOURCE_IMAGE can only be used when TEST_RELEASE_TYPE==Pre-GA ($CATALOG_SOURCE_IMAGE)"
-    exit 1
+    CATALOG_SOURCE_IMAGE="GA"
+    CATALOG_SOURCE_NAME="redhat-operators"
+    TRUSTEE_CATALOG_SOURCE_IMAGE="GA"
+    TRUSTEE_CATALOG_SOURCE_NAME="redhat-operators"
+    echo "Using default Red Hat operators catalog for GA release"
```

Comment:
This is basically what is changing, right? The first file is only about generating the workflows.

Now what this changes are the `catalogsourcename` and `trusteeCatalogSourcename` but it's not creating them (which is expected as they should exist).

The `CATALOG_SOURCE_IMAGE` and `TRUSTEE_CATALOG_SOURCE_IMAGE` are not used at all so it can be removed, right?

So my question is, what changed and how this helps with deployment? Is this about the openshift-test-private failing with empty `catalogsourcename`?

Reply:
Any prowjob on release/GA fails because line 75 was exiting if either SOURCE_IMAGE was set.